patches/server/0013-Paper-Plugins.patch

```diff
@@ -483,7 +486,37 @@ index 0000000000000000000000000000000000000000..39e503f52684e928c61c6d27bf61727a
 +        // We have to load the bukkit configuration inorder to get the update folder location.
 +        File configFileLocationBukkit = (File) minecraftOptionSet.valueOf("bukkit-settings");
 +        this.pluginDirectory = ((File) minecraftOptionSet.valueOf("plugins")).toPath();
-+        this.updateDirectory = this.pluginDirectory.resolve(YamlConfiguration.loadConfiguration(configFileLocationBukkit).getString("settings.update-folder", "update"));
++
++        String updateDirectory = YamlConfiguration.loadConfiguration(configFileLocationBukkit).getString("settings.update-folder", "update");
++        if (updateDirectory.isBlank()) {
++            this.updateDirectory = null;
++        } else {
++            Path resolvedUpdateDirectory = this.pluginDirectory.resolve(updateDirectory);
++            if (!Files.isDirectory(resolvedUpdateDirectory)) {
++                this.updateDirectory = null;
```

Comment:
Quietly disabling the update mechanism here is a bit meh, should log a warning since obviously the admin did something wrong

Reply:
The issue with this is that it includes if the directory is not there. That is the current behavior for that, at least.

Reply:
Oh right, I was thinking about the case where it's a file, not a folder, but yours is obviously more likely.
Is there any reason we shouldn't create the folder? 
Would be out of scope here I guess but generally help with discovery of this feature.

Reply:
Reason for not making a file is because that at least has been the default logic for a while... and you know.. people go crazy over stuff like that.

Reply:
Right, people are obsessed with their layouts and stuff, still something we could keep in mind for the future, discoverability for stuff like this warrents a change like this